### ISSUE TITLE
extension no longer starts with vscode as of 0.1.0

### ISSUE THREAD

**Issue by DetachHead** [microsoft/playwright-vscode#135]
re-raising #92 (see https://github.com/microsoft/playwright-vscode/issues/92#issuecomment-1042415286)

i think it's still an issue that the extension does not automatically start with vscode

**Comment by pavelfeldman** [comment]
VS Code extensions start upon certain activation events, but it should not be observable by the user. Something is different in your system set up because we can't reproduce it. Could you provide exact steps and env details for us to be able to try to reproduce?

**Comment by DetachHead** [comment]
sorry i'm not sure what additional info i can provide other than what i put in https://github.com/microsoft/playwright-vscode/issues/92#issuecomment-1041228706 and https://github.com/microsoft/playwright-vscode/issues/92#issuecomment-1042415286.

the exact steps to reproduce i don't think are very insightful:
1. press start
2. type vscode and hit enter
3. open the project (i usually click it from the recents section on the splash screen)
4. notice that the "Testing" tab isn't present

a couple facts about the machine i'm using off the top of my head, not sure if they are relevant:
- i'm behind an authenticated corporate proxy
- it has mcafee antivirus installed & running and i don't have access to turn it off

**Comment by mxschmitt** [comment]
@DetachHead do you have a playwright.config.ts present? Could you try creating a new version with "npm init playwright@latest foo"?

**Comment by DetachHead** [comment]
> do you have a playwright.config.ts present?

```ts
import type { PlaywrightTestConfig } from '@playwright/test';
import { devices } from '@playwright/test';

/**
 * Read environment variables from file.
 * https://github.com/motdotla/dotenv
 */
// require('dotenv').config();

/**
 * See https://playwright.dev/docs/test-configuration.
 */
const config: PlaywrightTestConfig = {
  testDir: './tests',
  /* Maximum time one test can run for. */
  timeout: 30 * 1000,
  expect: {
    /**
     * Maximum time expect() should wait for the condition to be met.
     * For example in `await expect(locator).toHaveText();`
     */
    timeout: 5000
  },
  /* Fail the build on CI if you accidentally left test.only in the source code. */
  forbidOnly: !!process.env.CI,
  /* Retry on CI only */
  retries: process.env.CI ? 2 : 0,
  /* Opt out of parallel tests on CI. */
  workers: process.env.CI ? 1 : undefined,
  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
  reporter: 'html',
  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
  use: {
    /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
    actionTimeout: 0,
    /* Base URL to use in actions like `await page.goto('/')`. */
    // baseURL: 'http://localhost:3000',

    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
    trace: 'on-first-retry',
  },

  /* Configure projects for major browsers */
  projects: [
    {
      name: 'chromium',
      use: {
        ...devices['Desktop Chrome'],
      },
    },

    {
      name: 'firefox',
      use: {
        ...devices['Desktop Firefox'],
      },
    },

    {
      name: 'webkit',
      use: {
        ...devices['Desktop Safari'],
      },
    },

    /* Test against mobile viewports. */
    // {
    //   name: 'Mobile Chrome',
    //   use: {
    //     ...devices['Pixel 5'],
    //   },
    // },
    // {
    //   name: 'Mobile Safari',
    //   use: {
    //     ...devices['iPhone 12'],
    //   },
    // },

    /* Test against branded browsers. */
    // {
    //   name: 'Microsoft Edge',
    //   use: {
    //     channel: 'msedge',
    //   },
    // },
    // {
    //   name: 'Google Chrome',
    //   use: {
    //     channel: 'chrome',
    //   },
    // },
  ],

  /* Folder for test artifacts such as screenshots, videos, traces, etc. */
  // outputDir: 'test-results/',

  /* Run your local dev server before starting the tests */
  // webServer: {
  //   command: 'npm run start',
  //   port: 3000,
  // },
};

export default config;
```

> Could you try creating a new version with "npm init playwright@latest foo"?

same issue. though interestingly, the first time i opened that newly created project the playwright extension did load on launch. however when i closed it and re-opened it again, it didn't

**Comment by pavelfeldman** [comment]
We did not see any upvotes for this for a substantial amount of time, which suggests that the problem might be on your side. We can't reproduce it outselves.

**Comment by LadaVarga** [comment]
I can repro on my side too:
**STR**
1. Have PlaywrightVscode installed and open the some playwright test or project.
2. Switch to the branch where you don't have any playwright test.
3. Open vscode.
4. PW Test button is not here (it is ok)
5. Switch back to branch with some PW tests.
6. Wait minute.

**Actual:** PW tests button is not here.
**Expected** It should be there.

Workaround: Disable and enable PlaywrightVSCode extension.
![2022-06-07 20_39_36-staging spec ts - esticom-frontend - Visual Studio Code](https://user-images.githubusercontent.com/22859476/172458410-5f1a12af-68e6-48d8-9385-78022b359972.png)
![2022-06-07 20_36_59-staging spec ts - esticom-frontend - Visual Studio Code](https://user-images.githubusercontent.com/22859476/172458413-96aab13c-f90d-48e2-ba4e-5aac5bdc3282.png)

**Comment by mxschmitt** [comment]
Closing as part of the triage process since it seemed stale or did not get enough upvotes in the past. Please create a new issue with a detailed reproducible (GitHub repository) or feature request if you still face issues.
